locale/en/download/package-manager.md

```diff
@@ -289,6 +290,28 @@ Or install a binary package (if available for your platform) using pkgin:
 pkgin -y install nodejs
 ```
 
+## nvm
+On Unix / OS X systems Node.js built from source can be installed by
+[nvm](https://github.com/creationix/nvm) by installing into the location that nvm expects:
+
+```bash
+$ make install DESTDIR='$(NVM_DIR)/versions/node/' PREFIX='$(FULLVERSION)-pre'
```

Comment:
please use `nvm_version_path "$VERSION"` so as to not hardcode the path - i'm much more likely to break the path than I am to break that internal `nvm` function.